app/api/payments/mpesa/route.ts

```
@@ -34,26 +35,41 @@
-    const authResponse = await fetch(
-      "https://sandbox.safaricom.co.ke/oauth/v1/generate?grant_type=client_credentials",
-      {
+    const authUrl =
+      "https://api.safaricom.co.ke/oauth/v1/generate?grant_type=client_credentials";
+    const authHeader = `Basic ${Buffer.from(`${process.env.MPESA_CONSUMER_KEY}:${process.env.MPESA_CONSUMER_SECRET}`).toString("base64")}`;
+
+    console.log("Attempting to fetch M-Pesa access token...");
+    console.log("URL:", authUrl);
+    console.log(
+      "Authorization Header (masked):",
+      authHeader.substring(0, 20) + "...",
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)